From: Chaitanya Kulkarni <kch@nvidia.com>
Date: Tue, 22 Feb 2022 19:36:56 -0800
Subject: nvme-tcp: don't initialize ret variable
Patch-mainline: v5.18-rc1
Git-commit: 462b8b2d84975758e5b74fe832bfe8145eef403f
References: jsc#PED-1183

No point in initializing ret variable to 0 in nvme_tcp_start_io_queue()
since it gets overwritten by a call to nvme_tcp_start_queue().

Signed-off-by: Chaitanya Kulkarni <kch@nvidia.com>
Reviewed-by: Sagi Grimberg <sagi@grimberg.me>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Acked-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/nvme/host/tcp.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/nvme/host/tcp.c
+++ b/drivers/nvme/host/tcp.c
@@ -1716,7 +1716,7 @@ static void nvme_tcp_stop_io_queues(stru
 
 static int nvme_tcp_start_io_queues(struct nvme_ctrl *ctrl)
 {
-	int i, ret = 0;
+	int i, ret;
 
 	for (i = 1; i < ctrl->queue_count; i++) {
 		ret = nvme_tcp_start_queue(ctrl, i);
